src/App.js

```diff
@@ -1,28 +1,18 @@
-import logo from './logo.svg';
+import { Routes, Route } from 'react-router-dom';
+import Navbar from './components/Navbar';
+import Countries from './components/Countries';
+import Country from './components/Country';
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Countries />} />
+        <Route path="/country/:id" element={<Country />} />
+      </Routes>
+    </>
   );
 }
 
```

Comment:
## App.test.js

- The test case in this test file is failing. please, double-check to fix it

-  [ ]  According to the project requirements :

   - The App should have unit tests for several components of the app and some `pure functions in Redux code` (currently you only tested the navbar, country component, and App). Please, kindly add that 🙏